cmd/allocator/v1alpha1/allocation.pb.go

```diff
@@ -0,0 +1,539 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
This will need a licence header as well. We have examples in the sdk gen tooling on how we did this.

Reply:
> Looks like this issue is the linter is failing on the generated code. I think you will need to have a line "This code was autogenerated. Do not edit directly." - which is included in https://github.com/googleforgames/agones/blob/master/build/boilerplate.go.txt
> 
> May be easier to get all the make commands working in this PR?

Thanks, yes I am working on the make commands. Will update this PR when done.